Merged archiver config (#4973)
camunda/camunda-docs: docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

Configure the exporter by providing `args`. See the tables below for configuration options and default values, or review the [example YAML configuration](#example).

| Option       | Description                                                                                                       | Default |
| ------------ | ----------------------------------------------------------------------------------------------------------------- | ------- |
| connect      | Refer to [Connect](./camunda-exporter.md?configuration=connect#options) for the connection configuration options. |         |
| index        | Refer to [Index](./camunda-exporter.md?configuration=index#options) for the index configuration options.          |         |
| bulk         | Refer to [Bulk](./camunda-exporter.md?configuration=bulk#options) for the bulk configuration options.             |         |

> | archiver     | Refer to [Archiver](./camunda-exporter.md?configuration=archiver#options) for the retention configuration options |         |

Reviewer: ```suggestion
| archiver     | Refer to [Archiver](./camunda-exporter.md?configuration=archiver#options) for the retention configuration options. |         |
```

`nanos`.
:::

</TabItem>

<TabItem value="archiver">

> To keep the main runtime index performant, the archiver will periodically move documents into historical

Reviewer: ```suggestion
To keep the main runtime index performant, the archiver periodically moves documents into historical
```
Suggestion: Use active voice

indices. The archiver can be configured as follows:

| Option                    | Description                                                                                                                                                                                                   | Default |
| ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |

> | enabled                   | If `true` archiver is enabled                                                                                                                                                                                 | `true`  |

Reviewer: ```suggestion
| enabled                   | If `true` archiver is enabled.                                                                                                                                                                                 | `true`  |
```

| elsRolloverDateFormat     | Defines how the date values are formatted for historical indices using Java DateTimeFormatter syntax, if no format is specified the first date format specified in teh field mapping is used.                 | `date`  |
| rolloverInterval          | The time range for groups captured by the historical indices.                                                                                                                                                 | `1d`    |

> | rolloverBatchSize         | The max number of instances for each batch that is being archived                                                                                                                                             | `100`   |

Reviewer: ```suggestion
| rolloverBatchSize         | The max number of instances for each batch that is being archived.                                                                                                                                             | `100`   |
```

| waitPeriodBeforeArchiving | Grace period for completed process instances to not be processed by the archiver. For a value of 1 hour, any process instances finished within the last hour will not be collected into the historical index. | `1h`    |
| delayBetweenRuns          | Time in milliseconds between archiving completed process instances.                                                                                                                                           | `2000`  |

> | maxDelayBetweenRuns       | Maximum time in milliseconds before an archiving attempt for completed process instances                                                                                                                      | `60000` |

Reviewer: ```suggestion
| maxDelayBetweenRuns       | Maximum time in milliseconds before an archiving attempt for completed process instances.                                                                                                                      | `60000` |
```

| retention                 | The configuration for a retention policy to delete old data.                                                                                                                                                  |         |

> 

Reviewer 2: ❓ Looks like we missing here some reference? Or should we embed them here?

Author: added reference to retention config
